File: store/src/app/shared/material.ts
export class Material {
    vId: number;
    vImage: string;
    vNameBuildingMaterial: string;
    vCatalog: string;
    vBrand: string;
    vPrice: number;

    constructor(vId: number, vImage: string, vNameBuildingMaterial: string, vCatalog: string,
      vBrand: string, vPrice: number) {
        this.vId = vId;
        this.vImage = vImage;
        this.vNameBuildingMaterial = vNameBuildingMaterial;
        this.vCatalog = vCatalog;
        this.vBrand = vBrand;
        this.vPrice = vPrice;
    }
}

File: Basket/Basket/Models/Material.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Linq;
using System.Threading.Tasks;

namespace Basket.Models
{
    public class Material
    {
        [Key]
        public int vId { get; set; }
        public string vImage { get; set; }
        public string vNameBuildingMaterial { get; set; }
        public string vCatalog { get; set; }
        public string vBrand { get; set; }
        public int vPrice { get; set; }
    }
}
